sdk/storage/azure-storage-file-datalake/swagger/README.md

```diff
@@ -16,7 +16,7 @@ autorest --v3 --python
 
 ### Settings
 ``` yaml
-input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/main/specification/storage/data-plane/Azure.Storage.Files.DataLake/preview/2021-06-08/DataLakeStorage.json
+input-file: https://github.com/seanmcc-msft/azure-rest-api-specs/blob/753cebf5af43d6bc402acc667b8def169111a838/specification/storage/data-plane/Azure.Storage.Files.DataLake/preview/2021-06-08/DataLakeStorage.json
```

Comment:
This can also be changed BUT it looks like it's not actually a new version. Both links are `2021-06-08` but I thought we are releasing `2021-08-06`? I'll check on this and get back to you.

Reply:
I have gone ahead and checked in the `main` directory, and it does seem that there is no new version. For now I will remove `- Added support for service version 2021-08-06.` from DataLake since it seems we have not added support for 08-06. I also regenerated pointing to main.

Reply:
Because of the change in `_serialize` adding the `2021-08-06` service version, we did actually add support for the new version despite not having a new generated version. Our clients will automatically take the latest version in `_serialize` instead of the generated version.

So, you should add that line back into the CHANGELOG :)

Reply:
Rats. I will go ahead and make that change again! At least CI is happy now 😄